package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.3",
   "description": "A sunrise/sunset/moonrise/moonset calculator",
   "main": "index.js",
+  "typings": "index.d.ts",
```

Comment:
Would you mind changed `typings` to `types`?

https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html